AppCenterDistribute/AppCenterDistribute/MSDistributePrivate.h

```diff
@@ -187,17 +187,6 @@ static NSString *const kMSTesterAppUpdateSetupFailedKey = @"MSTesterAppUpdateSet
  */
 - (void)disableAutomaticCheckForUpdate;
 
-/**
- * Check the latest release of the application.
- *
- * @param updateToken The update token stored in keychain. This value can be nil if it is public distribution.
- * @param distributionGroupId The distribution group Id in keychain.
- * @param releaseHash The release hash of the current version.
- */
-- (void)checkForUpdateWithUpdateToken:(nullable NSString *)updateToken
```

Comment:
This is a rollback as it was added earlier based on different understanding of the spec.